src/auth/roles/index.ts

```diff
@@ -0,0 +1,126 @@
+import { RolesInitialiser } from './types';
+const { lazyPromiseSeries } = require('../../../database');
```

Comment:
If this function is useful in more than one place now, it's probs worth sticking it in a utils file or something somewhere.